app/Config/bootstrap.default.php

```diff
@@ -107,6 +107,19 @@
 	}
 }
 
+/**
+ * Configure base URL for CakePHP
```

Comment:
Wouldn't you need to consider "MISP.external_baseurl" too?
/cc @iglocska

Reply:
External baseurl is simply for identifying the instance and has no bearing on the URLs.

Reply:
Being the never-ending-n00b here, in which scope is this?

Who/where is the identifying done?

Reply:
When you create a sharing group, your instance is included in the list by default. This is implicitly part of the SG. When you synchronise, MISP attaches your own server's identifier - before we used the baseurl, which lead to your own server not being represented for empty baseurls or your server being misrepresented for internal baseurl settings (for example, 192.168.1.50 for your MISP is great to access it internally, sucks for everyone else). This is why you can now set the baseurl to whatever you want to use to reach your MISP but include the externally visible URL via the external_baseurl for sharing group creation.

Reply:
Ok, I perhaps "understand" my confusion....

Because there is no clear separator (in the table of the MISP configuration explanation) I misread the thing.

UX ftw!

It was squeezed and I merged the tables... (in my brain, I guess)
Which make the explanation incomprehensible. Of course this might only attend to a minority, so no problem.

Reply:
No clear separator? o.O

![image](https://user-images.githubusercontent.com/3668672/58254580-54d92380-7d6b-11e9-9da7-ba0f5b3ee9cf.png)

Reply:
Well if the value is NOT set, you have the Error Message "blending" into the left hand table.